### PR TITLE
chore(README): Clearifications

### PR DESCRIPTION
**Pull request by doktordirk**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -98,43 +104,51 @@ The above configuration file can cope with a development and production version
...
+```javascript
+import authConfig from './authConfig';
+
+...
+
+aurelia.use
+  /* Your other plugins and init code */
+  .plugin('spoonx/aurelia-api', configure => {
+    configure
+      .registerEndpoint('auth',https://myapi.org/auth)
```
Comment (RWOverdijk) [inline README.md]: Won't work.

File: README.md [hunk 2]
```diff
@@ -98,42 +105,27 @@ The above configuration file can cope with a development and production version
...
 export function configure(aurelia) {
   aurelia.use
-    .standardConfiguration()
-    .developmentLogging()
-    .plugin('aurelia-animator-css')
-    .plugin('spoonx/aurelia-auth', (baseConfig)=>{
-         baseConfig.configure(config);
+    /* Your other plugins and init code */
+    .plugin('spoonx/aurelia-api', configure=>{
+      configure
```
Comment (RWOverdijk) [inline README.md:122]: I find this part confusing. So I need to register three api endpoints to use this module? I think it can be clearer what it's for?

Comment (doktordirk) [inline README.md:122]: isn't that what it's for? it's not van-blandels auth anymore. it's the simplest as well as  the indended way is to use it with aurelia-api. imho BC was just a nicety. before i got shortly lazy, i wanted to add samples for the other options in /doc, but they all need much more explanation

Comment (RWOverdijk) [inline README.md:122]: I agree completely, I just think we should describe why these options are added like this. Link to the api docs, or perhaps explain that the name of the endpoint is used in baseConfig.configure(authConfig);

**Pull request merged**